docs/REST API/rest-api.md

```diff
@@ -642,7 +642,13 @@ Retrieves a list of the non expired carves. Carve contents remain available for
 
 #### Parameters
 
-None.
+| Name            | Type    | In    | Description                                                                                                                   |
+|-----------------|---------|-------|-------------------------------------------------------------------------------------------------------------------------------|
+| page            | integer | query | Page number of the results to fetch.                                                                                          |
+| per_page        | integer | query | Results per page.                                                                                                             |
+| order_key       | string  | query | What to order results by. Can be any field listed in the `results` array example below.                                       |
+| order_direction | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
```

Comment:
Would wrap the possible options in quotes instead of backticks since this parameter's a string. Also, might update the description to use the "Valid options are..." language to make it consistent with the other updates you made.
```suggestion
| order_direction | string  | query | **Requires `order_key`**. The direction of the order given the order key. Valid options are 'asc' or 'desc'. Default is 'asc'. |
```